Userland/Libraries/LibGfx/PortableImageLoaderCommon.h

```diff
@@ -53,12 +52,13 @@ static inline ErrorOr<u16> read_number(Streamer& streamer)
 }
 
 template<typename TContext>
-static ErrorOr<void> read_comment([[maybe_unused]] TContext& context, Streamer& streamer)
+static ErrorOr<void> read_comment(TContext& context)
 {
+    auto& stream = *context.stream;
     bool is_first_char = true;
     u8 byte {};
 
-    while (streamer.read(byte)) {
+    while ((byte = TRY(stream.template read_value<u8>()))) {
```

Comment:
Unneccessary braces

Reply:
Not unnecessary braces
`serenity/Userland/Libraries/LibGfx/PortableImageLoaderCommon.h:61:17: error: suggest parentheses around assignment used as truth value [-Werror=parentheses]`